Userland/Libraries/LibArchive/Zip.cpp

```diff
@@ -90,7 +91,7 @@ ErrorOr<bool> Zip::for_each_member(Function<IterationDecision(ZipMember const&)>
         member.compression_method = central_directory_record.compression_method;
         member.uncompressed_size = central_directory_record.uncompressed_size;
         member.crc32 = central_directory_record.crc32;
-        member.is_directory = central_directory_record.external_attributes & zip_directory_external_attribute || member.name.bytes_as_string_view().ends_with('/'); // FIXME: better directory detection
+        member.is_directory = central_directory_record.external_attributes & zip_directory_external_attribute || Core::File::is_directory(member.name.to_deprecated_string());
```

Comment:
A directory existing on-disk that (by chance) has the same name as the current member doesn't automatically make the current member a directory. This presumably also makes parsing a lot slower, since we are accessing the disk for each member.

Reply:
Okay, will remove this change